docs/USAGE-GUIDE.md

```diff
@@ -486,6 +486,30 @@ To use certificate transparency, the requester (usually the client) must call `s
 
 Call `s2n_connection_get_sct_list()` to retrieve the received certificate transparency information. The format of this data is the SignedCertificateTimestampList structure defined in section 3.3 of RFC 6962.
 
+## Session Resumption
+
+TLS handshake sessions are CPU-heavy due to the calculations involved in authenticating a certificate. These calculations can be skipped after the first connection by turning on session resumption. This mechanism stores state from the previous session and uses it to establish the next session, allowing the handshake to skip the costly authentication step while keeping the same cryptographic guarantees. Note that this session state is encrypted by the server, so a server will have to set up an external key in order to do session resumption.
+
+### Stateless Session Resumption
+
+In stateless session resumption the server creates a 'session ticket' from some state after a successful handshake. The server sends this ticket to a client, and the client can send that ticket back to the server during a new connection to skip the authentication step. This mechanism allows servers to avoid storing individual state for each client, and for that reason is the preferred method for resuming a session.
+
```

Comment:
Do you want to call out how this is safe? Readers might go "but how do I know the ticket is valid / real."

Reply:
Can you elaborate on what you mean here? Who are we talking about, the server or the client?

Reply:
The server I guess. I meant we might want to at least briefly mention why session tickets can replace certificates for auth.